scripts/upgrade

```diff
@@ -76,7 +76,8 @@ ynh_install_php7
 
 # Download, check integrity, uncompress and patch the source from app.src
 ynh_setup_source "$final_path"
-
+init_composer $final_path
+cd $final_path && sudo /usr/bin/php7.2 bin/console doctrine:migrations:migrate
```

Comment:
Can you check for the previous version?
You should also clear and warmup the cache in the context (with the permissions) of the webserver user.

Reply:
I have added  "bin/console cache:clear --env=prod"  and " bin/console cache:warmup --env=prod" as mentioned in the document https://github.com/kevinpapst/kimai2/blob/master/UPGRADING.md